app/views/schedules/new.html.erb

```diff
@@ -4,8 +4,8 @@
 <div class="row">
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
-        <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
-        <%= button_tag 'Add', type: :submit, class: 'btn btn-sm btn-primary' %>
+        <%= render partial: 'fields', locals: {f: f, object: @schedule } %>
+        <%= button_tag 'Add Root Task', type: :submit, class: 'btn btn-sm btn-primary' %>
```

Comment:
Since I haven't figured out a way to block the addition of multiple tasks before the root task is saved (and all non-root tasks require a parent task), I phrased it that way to try to remind the user to enter exactly one root task before adding other tasks. It's an easy change back however if you want.